src/modules/hash.js

```diff
@@ -0,0 +1,22 @@
+import SHA1 from 'crypto-js/sha1'
+import { DIMENSION_TYPE_EXPRESSION_DIMENSION_ITEM } from './dataTypes.js'
+import { layoutGetAllItems } from './layout/layoutGetAllItems.js'
+
+const isValidValue = (value) => typeof value === 'string' && value.length
+
+export const getHash = (value) =>
```

Comment:
Is this exported for the sake of jest tests? Personally I would prefer moving those test scenarios to the tests for `getExpressionHashFromVisualization` and not export the getHash function, to keep things cleaner.

Reply:
@janhenrikoverland

Reply:
@jenniferarnesen  We might want to swap out the SHA1 for something else, so I think it's good to have a couple of tests for the hash function directly. If the new function returns something unexpected, the problem isn't hidden somewhere in our dimension structure.